posixlib/src/main/scala/scala/scalanative/posix/time.scala

```diff
@@ -10,7 +10,17 @@ object time {
   type time_t   = types.time_t
   type clock_t  = types.clock_t
   type timespec = CStruct2[time_t, CLong]
-  type tm       = CStruct9[CInt, CInt, CInt, CInt, CInt, CInt, CInt, CInt, CInt]
+  type tm = CStruct11[CInt,
```

Comment:
I think something like the block below would add some context for future
developers.
```
type timespec = CStruct2[time_t, CLong]

// POSIX X/Open System Interface (XSI) extension
 type tm = CStruct11[CInt,
```